.github/workflows/actions.yml

```diff
@@ -0,0 +1,24 @@
+---
+name: GitHub Actions
+
+# yamllint disable-line rule:truthy
```

Comment:
As there is no yamllint configured to run, I think this comment shouldn't be here